Nodejs/Product/Nodejs/NodejsProject.cs

```diff
@@ -544,7 +544,7 @@ out project
             var properties = GetExtensionObject(_innerVsHierarchy, selectionItemId).Properties;
             try {
                 var itemType = properties.Item("ItemType").Value;
-                ourEditor = (string)itemType == ProjectFileConstants.Compile && !IsES6IntellisensePreview() ? Guids.NodejsEditorFactory : Guid.Empty;
+                ourEditor = Guid.Empty;
             } catch (ArgumentException) {
                 // no item type, file is excluded from project.
                 ourEditor = Guids.NodejsEditorFactory;
```

Comment:
I've got another change that builds on this one to remove the node editors entirely (you can still reach them using `Open With`). I'll make sure this function is updated as part of that work.